scripts/infobase-server

```diff
@@ -5,11 +5,11 @@ USAGE:
 
 * Run Infobase http server at port 7070.
 
-    $ python ./scripts/infobase-server infobase.yaml 7070
+    $ python ./scripts/infobase-server infobase.yml 7070
```

Comment:
Good catch on the file names here. I am not sure whether this port is 7070 or 7000. This will take more investigation.

Reply:
Please leave to port numbers unmodified!

Reply:
The thing that confused me here is that the comments mention port 7070, but what's now `docker-compose.yml` starts infobase with `command: docker/ol-infobase-start.sh`, and `docker/ol-infobase-start.sh` calls this script with port 7000, rather than 7070.

Port 7000 is also the port that is exposed for infobase in what's currently `docker-compose.yml`, and it's the port the infobase container is listening on when started via these compose files.

I grepped the codebase for `7070`, but the only references seem to be comments, but I know there may be more at play here than I am able to appreciate.